fb.c

```diff
@@ -299,12 +324,10 @@ int vcamfb_init(struct vcam_device *dev)
     vfb_fix.line_length = dev->input_format.bytesperline;
 
     /* set the fb_var */
+    vfb_default.xres = dev->input_format.width;
+    vfb_default.yres = dev->input_format.height;
     if (dev->conv_crop_on) {
-        vfb_default.xres = dev->crop_output_format.width;
-        vfb_default.yres = dev->crop_output_format.height;
-    } else {
-        vfb_default.xres = dev->input_format.width;
-        vfb_default.yres = dev->input_format.height;
+        set_crop_resolution(&vfb_default.xres, &vfb_default.yres);
     }
```

Comment:
I think currently this is correct to  assign `crop_output_format` resolution to `vfb_default` in `vcamfb_init`
But I have considered three points in this section:
1. In this version, I want to emphasis that `fb` should strongly related to `input_format` but not `ouput_format` including `crop_out_fmt`. 
2. For sure we can assign it right away because they have the same value when initialize, but there is a possibility that we might change the implementation or use `vcamfb_init` in other scene, and it is better to reduce unnecessary dependency.
3. I want to unite the coding style from `fb.c #416`.

Reply:
I recommend to assign from `crop_output_format ` two reasons.
1. Calling function still has cost.
2. `vcamfb_init` just for init, only called from `create_vcam_device`.

Reply:
Thanks, I think I will change this back to origin